examples/resgen/Strings.Designer.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Hello {

```

Comment:
Should this have been committed?

Reply:
The `.Designer.cs` file? Yes. Without it you would just get a compilation error about `Strings` not existing in the context.

You can generate this file in the compilation process using a `Compile` task on the designer file (e.g. `<Compile Update="Strings.Designer.cs">`).